### PR TITLE
git-annex 6.20181011

### PR DESCRIPTION
**Pull request by commitay**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by fxcoudert** [comment]
Let's remove the never-used option.

**Comment by commitay** [comment]
> Let's remove the never-used option.

👍 

> Can we remove the version pinning?

No.

> Does it build with latest `ghc`?

No, because of the version pinning.

**Comment by commitay** [comment]
>  depends_on "xdot" => :recommended

could make this required.

File: Formula/git-annex.rb [hunk 1]
```diff
@@ -34,12 +32,6 @@ def install
                           "--constraint", "network<2.7.0.1",
                           :using => ["alex", "happy", "c2hs"],
                           :flags => ["s3", "webapp"] do
```
Comment (commitay) [inline Formula/git-annex.rb]: I think `do ... end` isn't needed now.

**Comment by commitay** [comment]
@BrewTestBot test this please

**Pull request closed without merging**